test/functional/ui/float_spec.lua

```diff
@@ -7947,7 +7947,7 @@ describe('float window', function()
         qui officia deserunt mollit anim id est
         laborum.]])
       local buf = api.nvim_create_buf(false,false)
-      api.nvim_buf_set_lines(buf, 0, -1, true, {"test", "", "popup    text"})
+      api.nvim_buf_set_lines(buf, 0, -1, true, {"test\u{2800}", "", "popup\u{2800}   text"})
```

Comment:
I think the puc-lua CI job is failing because `\u` is a luajit-only feature. try this (byte literals instead of unicode codepoint) (I didn't verify the bytes, this is from AI):

```suggestion
      local braille_blank = "\226\160\128"
      api.nvim_buf_set_lines(buf, 0, -1, true, {"test" .. braille_blank, "", "popup"..braille_blank.."   text"})
```

Reply:
I would also recommend restricting this to a single test instead of modifying all the tests; that makes isolating failures easier. (Or just copy the simplest one and add the non-space blank.)

Reply:
Ohhhh, that makes a lot more sense
The result indeed seemed weird to me
I just wanted to run this once more through CI to make sure it wasn't just github acting weird